python/README.md

```diff
@@ -89,38 +99,56 @@ cos = ibmos2spark.CloudObjectStorage(sc, credentials,
                                       cos_type='bluemix_cos',
                                       auth_method='iam_token')
 
+# The `cos` object will provide the URL for SparkContext to retrieve your data
+# NOTE the counter-intuitive order of 'object_name' followed by 'bucket_name' here
+
 bucket_name = 'bucket_name'
 object_name = 'file_name'
-data = sc.textFile(cos.url(object_name, bucket_name))
+data_url = cos.url(object_name, bucket_name)
+
+data = sc.textFile(data_url)
 ```
 
 
-### Bluemix Swift Object Storage / Data Science Experience
+### Cloud Object Storage (IaaS)
+
+This is the service described **middle left** pane in the image above. This service is sometimes refered to 
+as the Softlayer IBM Cloud Object Storage service.
+
 
 ```python
 import ibmos2spark
 
-#To obtain these credentials in IBM Spark, click the "insert to code"
-#button below your data source found on the panel to the right of your notebook.
-
 credentials = {
-  'auth_url': 'https://identity.open.softlayer.com',  #your URL might be different
-  'project_id': '',
-  'region': '',
-  'user_id': '',
-  'username': '',
-  'password': '',
+  'endpoint': 'https://s3-api.objectstorage.softlayer.net/',  #just an example. Your url might be different
+  'access_key': '',
+  'secret_key': ''
 }
 
-configuration_name = 'my_bluemix_os'  #you can give any name you like
+configuration_name = 'cos_config_string'  #you can give any string you like
+cos = ibmos2spark.CloudObjectStorage(sc, credentials, configuration_name=configuration_name)  #sc is the SparkContext instance.
 
-bmos = ibmos2spark.bluemix(sc, credentials, configuration_name)  #sc is the SparkContext instance
+# The `cos` object will provide the URL for SparkContext to retrieve your data
+# NOTE the counter-intuitive order of 'object_name' followed by 'bucket_name' here
+
+bucket_name = 'bucket_name'
+object_name = 'file_name'
+data_url = cos.url(object_name, bucket_name)
+
+data = sc.textFile(data_url)
 
-data = sc.textFile(bmos.url(container_name, object_name))
 ```
 
 
-### Softlayer Swift Object Storage
+
+### Object Storage OpenStack Swift (Iaas)
+
+This is the service described in **middle right** pane in the image above (and was previously referred to 
+as Softlayer Swift Object Storage). 
+
+Note below that credentials are not passed in as a dictionary, like in the other implementations. 
+Rather, each piece of information is supplied as a separate, required arguement. 
+Also, the `CloudObjectStorage` class is not used here, but rather a `softlayer` object is instantiated.
```

Comment:
This line is a bit confusing. We don't need to mention CloudObjectStorage here

Reply:
removed